policyengine_us/parameters/gov/states/pa/dhs/tanf/maximum_asset_value.yaml

```diff
@@ -0,0 +1,9 @@
+description: Pennsylvania allows for the following maximum assets as a TANF eligibility criteria.
+values:
+  2021-01-01: 1_000
+metadata:
+  unit: currency-USD
+  label: Pennsylvia TANF maximum assets
+  reference:
+    - title: Depertment of Human Services - Temporary Assistance for Needy Families (TANF) and Moving to Independence
```

Comment:
Please use the following source:
http://services.dpw.state.pa.us/oimpolicymanuals/cash/index.htm#t=140_Resources%2F140_1_General_Policy.htm&rhsearch=1%2C000&rhsyns=%20

Reply:
Let's actually use the legal code: https://www.pacodeandbulletin.gov/Display/pacode?file=/secure/pacode/data/055/chapter177/s177.31.html&d=reduce

You can add it as a second reference. But title this one `140.11 Resource Limits` as the relevant section of the program book.